src/__tests__/threeid-provider.test.ts

```diff
@@ -115,7 +111,6 @@ describe('ThreeIdProvider', () => {
 
   beforeAll(async () => {
     tmpFolder = await tmp.dir({ unsafeCleanup: true })
-    // @ts-ignore
     ipfs = await Ipfs.create(genIpfsConf(tmpFolder.path))
     ceramic = await Ceramic.create(ipfs, {
```

Comment:
We should really switch to using the ceramic jest environment at some point soon.